control-plane/subcommand/server-acl-init/anonymous_token.go

```diff
@@ -44,3 +51,26 @@ func (c *Command) configureAnonymousPolicy(consulClient *api.Client) error {
 			return err
 		})
 }
+
+func checkIfAnonymousTokenPolicyExists(consulClient *api.Client) (bool, error) {
+	token, _, err := consulClient.ACL().TokenRead("00000000-0000-0000-0000-000000000002", nil)
```

Comment:
You could move the anonymous token ID as a constant since it is being referenced at other places in this file